fluent/toInterfaceValue.go

```diff
@@ -0,0 +1,67 @@
+package fluent
+
+import (
+	"github.com/ipld/go-ipld-prime/datamodel"
+)
+
+func ToInterfaceValue(node datamodel.Node) (interface{}, error) {
+	switch k := node.Kind(); k {
+	case datamodel.Kind_Invalid:
+		panic("invalid node")
```

Comment:
Panicking here seems a bit harsh :) An error woudl be better. Otherwise one would have to wrap most calls to this func with a recover, because it's hard to tell in advance if some input data or edge case program behavior could lead to an invalid kind.